### PR TITLE
Choi cutoff fix

### PR DESCRIPTION
**Pull request by ziofil**
**Context:**
the choi method sometimes uses wrong cutoffs

**Description of the Change:**
fixes it by picking the cutoffs from the input state

**Benefits:**
correct implementation

**Possible Drawbacks:**
none

**Related GitHub Issues:**
none

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -222,6 +222,9 @@
 * Fixed a bug in the Gaussian state which caused its covariance matrix to be multiplied
   by hbar/2 twice. Adds the argument `modes` to `Ggate`.
   [(#212)](https://github.com/XanaduAI/MrMustard/pull/212)
+]
```
Comment (sduquemesa) [inline .github/CHANGELOG.md:225]:
```suggestion

```

**Review by sduquemesa — APPROVED** [review]
🐛🔨

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -222,6 +222,8 @@
 * Fixed a bug in the Gaussian state which caused its covariance matrix to be multiplied
   by hbar/2 twice. Adds the argument `modes` to `Ggate`.
   [(#212)](https://github.com/XanaduAI/MrMustard/pull/212)
+* Fixes a but in the cutoffs of the choi operator.
```
Comment (sduquemesa) [inline .github/CHANGELOG.md:225]:
```suggestion

* Fixes a but in the cutoffs of the choi operator.
```

File: .github/CHANGELOG.md [hunk 3]
```diff
@@ -223,6 +223,9 @@
   by hbar/2 twice. Adds the argument `modes` to `Ggate`.
   [(#212)](https://github.com/XanaduAI/MrMustard/pull/212)
 
+* Fixes a but in the cutoffs of the choi operator.
```
Comment (ziofil) [inline .github/CHANGELOG.md:226]:
```suggestion
* Fixes a bug in the cutoffs of the choi operator.
```

**Pull request merged**